page-objects/avatarDropdownMenu.ts

```diff
@@ -0,0 +1,17 @@
+import { Locator, Page } from "@playwright/test";
+import { logClickingOnElement } from "../utils/decorators";
+
+export class AvatarDropdownMenu {
+  private readonly page: Page;
+  private readonly profile: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.profile = this.page.getByTestId("dropdown-list-item__user");
+  }
+
+  @logClickingOnElement("'Profile' menu item")
+  async clickSettings() {

```

Comment:
Metoda ta otwiera "Profile", opcja "Settings" jest niżej i ma inny lokator (`dropdown-list-item__mySettings`), co może być trochę mylące. Można zmienić na:
```suggestion
  async clickProfile() {
```
lub
```suggestion
  async clickProfileOption() {
```